hw3/01-characters.js

```diff
@@ -1,4 +1,63 @@
+const selectSection = document.querySelector("section");
+
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
+
+fetch(url)
+	.then((Response) => {
+		return Response.json();
+	})
+	.then((data) => {
+		data.forEach((item) => {
+			let card = document.createElement("div");
```

Comment:
Seems like the most efficeint way of doing it, and required as such by the professor. But IMO, I like it as using the inner html with template litterals for the stuff. Just a side note